benchmarks/run_all.dart

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2021, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+import 'dart:math';
+
+import 'latency.dart';
+import 'throughput.dart';
+
+void main(List<String> args) async {
+  var url = 'https://example.com';
+  var throughputPrallelLimit = 10;
+  switch (args.length) {
```

Comment:
We can also _consider_ using `package:args` throughout the package for handling cmd line args. But that's probably good for later if we end up with more arguments.

Reply:
Yeah. Let's use `package:args` in other places too.

Reply:
In a separate PR maybe to keep the commit history more meaningful? As, I'll have to migrate other parts of the package too. (i.e. `bin/setup.dart`).